packages/terra-popup/tests/wdio/popup-spec.js

```diff
@@ -0,0 +1,16 @@
+/* global  browser,Terra */
+const viewports = Terra.viewports('tiny', 'medium');
+
+describe('Popup', () => {
+  describe('Arrow Popup', () => {
+    beforeEach(() => browser.url('/#/tests/popup-tests/arrow'));
+
+    Terra.should.beAccessible({ viewports });
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.themeEachCustomProperty({
+      '--terra-popup-content-rounded-corners-border-radius': '20px',
+      '--terra-popup-content-inner-box-shadow': '10px 10px 36px blue',
+      '--terra-popup-content-inner-background-color': 'yellow',
```

Comment:
1) You have a screenshot for --terra-popup-inner-border, but you don't have a test for it here, that screenshot should probably be removed.
2) You no longer have the themeing variable --terra-popup-content-rounded-corners-border-radius, you should remove that from the test and the screenshot.
3) Your screenshots for the viewport tests don't have the popup open, so that's not testing what we would want tested here.

Reply:
Number 3 is very weird. We added `      browser.waitForVisible('[data-terra-popup-content]');
` to the tests, and it still didn't show anything. We will keep looking